### PR TITLE
Display AZs in gsctl show cluster

### PR DESCRIPTION
**Pull request by MarcelMue**
This shows the list of AZs a cluster is running on inside the `gsctl show cluster` command.
towards giantswarm/giantswarm#2202

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -39,7 +39,7 @@
   version = "=2.0.2"
 
 [[constraint]]
-  branch = "master"
+  branch = "add-az-status"
```
Comment (MarcelMue) [inline Gopkg.toml]: TODO: revert

File: commands/show_cluster.go [hunk 2]
```diff
@@ -250,6 +250,10 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 
 	output = append(output, color.YellowString("Kubernetes API endpoint:")+"|"+clusterDetails.APIEndpoint)
 
+	if len(clusterDetails.AvailabilityZones) > 0 {
+		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ","))
```
Comment (MarcelMue) [inline commands/show_cluster.go]: Pretty straight forward, but no in which order we want this magical output to be.

**Comment by MarcelMue** [comment]
Tested on gauss:
```
gsctl show cluster j64f6                         
ID:                        j64f6
Name:                      Timo: k8s 1.12 multiaz
Created:                   2018 Nov 23, 11:43 UTC
Organization:              giantswarm
Kubernetes API endpoint:   https://api.j64f6.k8s.gauss.eu-central-1.aws.gigantic.io
Availability Zones:        eu-central-1a,eu-central-1b
Release version:           6.1.0
Workers:                   4
Worker instance type:      m5.large
CPU cores in workers:      8
RAM in worker nodes (GB):  32.00
```
And with one AZ:
```
gsctl show cluster zy6ya
ID:                        zy6ya
Name:                      marcel-test
Created:                   2018 Nov 23, 09:03 UTC
Organization:              giantswarm
Kubernetes API endpoint:   https://api.zy6ya.k8s.gauss.eu-central-1.aws.gigantic.io
Availability Zones:        eu-central-1a
Release version:           6.1.0
Workers:                   3
Worker instance type:      m3.large
CPU cores in workers:      6
RAM in worker nodes (GB):  22.50
```

**Review by tuommaki — APPROVED** [review]
Maybe sort AZs alphabetically. Otherwise all good here, given it works when tested :slightly_smiling_face:

File: commands/show_cluster.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (MarcelMue) [reply commands/show_cluster.go]: Done :+1:

File: commands/show_cluster.go [hunk 3]
```diff
@@ -250,6 +251,11 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 
 	output = append(output, color.YellowString("Kubernetes API endpoint:")+"|"+clusterDetails.APIEndpoint)
 
+	if len(clusterDetails.AvailabilityZones) > 0 {
+		sort.Strings(clusterDetails.AvailabilityZones)
+		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ","))
```
Comment (marians) [inline commands/show_cluster.go]:
```suggestion
		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ", "))
```

Aligning this with instance types/vm sizes in the `gsctl info` command

**Review by marians — APPROVED** [review]

**Pull request merged**